app/models/microservices_engine/connection.rb

```diff
@@ -31,7 +31,7 @@ def get(path, params = {})
 
     def self.get(resource, path, params = {})
       conn = Connection.where(object: resource.to_s) # Does :abc match "abc"?
-      conn ? conn.get(path, params) : {}
+      conn.get(path, params) if conn
```

Comment:
Solely for readability, yes. It's closer to how you would express the idea in plain speech.